src/server.ts

```diff
@@ -21,11 +21,13 @@ mongoose
 
 // bring in routes
 import productRoutes from './routes/product';
+import authRoutes from './routes/auth';
 
 //middleware
```

Comment:
I don't know for sure but I think that the frontend won't be able to make requests unless it is authorized by the CORS middleware

Reply:
Okay good point i will implement a cors middleware.

Reply:
I will create a new pull request for that

Reply:
can you create an issue for that with your screenshot?